### PR TITLE
Account-Level Config -- Truncate Targeting Attribute 

### PR DESCRIPTION
**Pull request by AlexBVolcy**
This PR addresses this issue: https://github.com/prebid/prebid-server/issues/1266

Within the file `exchange/targeting.go`, there is a const int value `MaxKeyLength` that is utilized in this file's `addKeys` function to limit define the max length a bidder name can be. 

What this PR is doing, is allowing this value to be configurable at the account level, with the new variable being named `truncateTargetAttr` to match the PBS Java name. So if the `addKeys` function is called, it'll check if at the account level, `truncateTargetAttr` has been defined. If it has, then we use that value instead of the const `MaxKeyLength`, if it hasn't, then we just use `MaxKeyLength`.

The test `TestSetTargeting` has been updated to accommodate these changes.

### CODE REVIEW

File: exchange/targeting.go [hunk 1]
```diff
@@ -35,56 +35,62 @@ type targetData struct {
...
 			topBidPerBidder.bidTargets = targets
 		}
 	}
 }
 
-func (targData *targetData) addKeys(keys map[string]string, key openrtb_ext.TargetingKey, value string, bidderName openrtb_ext.BidderName, overallWinner bool) {
+func (targData *targetData) addKeys(keys map[string]string, key openrtb_ext.TargetingKey, value string, bidderName openrtb_ext.BidderName, overallWinner bool, truncateTargetAttr *int) {
+	var maxLength int
+	if truncateTargetAttr != nil {
+		maxLength = *truncateTargetAttr
```
[the remark this answers is not in the document]
Comment (VeronikaSolovei9) [reply exchange/targeting.go:88]: Should it be just `int`, not `*int`? If it's 0 - then consider it's not set and take default value.

**Review by hhhjort — APPROVED** [review]
LGTM

**Review by VeronikaSolovei9 — APPROVED** [review]
LGTM

**Review by hhhjort — APPROVED** [review]
LGTM

**Pull request merged**